File: codingsample/202008/20200825/Rains_of_Reason_5_chessBoardCellColor.c
#include <stdio.h>
#include <stdlib.h>
#include <stdbool.h>
#include <string.h> 

bool chessBoardCellColor(char * cell1, char * cell2) {
    return ((((cell1[0]-'A')%2)==(cell1[1]%2))==(((cell2[0]-'A')%2)==(cell2[1]%2)));
}


int main() {
	char* ce1 = "A1";
	char* ce2 = "C3";
	if(chessBoardCellColor(ce1,ce2)) printf("True\n");
	else printf("False\n");
}

File: codingsample/202008/20200825/Rains_of_Reason_5_chessBoardCellColor.py
def chessBoardCellColor(cell1, cell2):
        temp1 = list(cell1)
        temp2 = list(cell2)
        return (((ord(temp1[0])-ord('A')+1)%2)==(int(temp1[1])%2))==(((ord(temp2[0])-ord('A')+1)%2)==(int(temp2[1])%2))

print(chessBoardCellColor("A3","C1"))